packages/flutter/lib/src/gestures/pointer_signal_resolver.dart

```diff
@@ -99,14 +131,31 @@ class PointerSignalResolver {
       return;
     }
     assert(_isSameEvent(_currentEvent!, event));
+
+    print(event.kind);
+    // TODO(Ichordedionysos): Re-enable check for trackpad once the event.kind is properly exposing trackpad usage.
+    if (event is PointerScrollEvent /*&& event.kind == PointerDeviceKind.trackpad*/) {
```

Comment:
Accurate `event.kind` is only possible through my heuristic in certain browsers. But I think it's safe to lock axis on mouse scrolling too unconditionally, usually mouse scrolling is just one axis anyway.

Reply:
Yes, I had the same thought 👍